Info-Orbs/src/core/utils.cpp

```diff
@@ -1,6 +1,5 @@
 #include "utils.h"
 
-
 int Utils::getWrappedLines(String (&lines)[MAX_WRAPPED_LINES], String str, int limit) {
```

Comment:
It would be good to start thinking about unit-tests for code like this.

Reply:
I haven't written unit tests for C++ code but I agree they would be useful.